kpc/management/commands/load_certs.py

```diff
@@ -0,0 +1,337 @@
+from django.core.management.base import BaseCommand
+from django_countries import countries
+from django_countries.fields import Country
+
+from kpc.models import Certificate, Licensee, PortOfExport
+
+EXPECTED_DATE_FORMAT = '%Y-%m-%d %H:%M:%S.%f'
+
+# Map HScode to objects from initial-data.json
+HSCODE_MAP = {'3': 2, '4': 1, '6': 3}
+
+# Map status codes to Certificate attributes
+STATUS_MAP = {'2': Certificate.PREPARED,
+              '3': Certificate.DELIVERED, '4': Certificate.INTRANSIT}
+
+# relative path from manage.py to the refCountries.csv file
+COUNTRY_CSV = './data/refCountries.csv'
+
+
+class Command(BaseCommand):
+    help = 'Load certificate data from csv export'
+
+    def add_arguments(self, parser):
+        parser.add_argument('filepath', type=str)
+        parser.add_argument('--limit', dest='limit', nargs='?', type=int)
+
+    def handle(self, *args, **options):
+        filepath = options['filepath']
+        self.limit = options['limit']
+        if self.limit:
+            self.stdout.write(f'Limiting to {self.limit} rows.')
+
+        self.stdout.write(f'Reading data from {filepath}...')
+        self.rectified_csv = rectify_csv(filepath)
+        self.stdout.write("Newlines converted to '|'")
+
+        self.licensee_id_list = list(
+            Licensee.objects.all().values_list('id', flat=True))
+        self.port_of_export_map = {
+            poe.name: poe.id for poe in PortOfExport.objects.all()}
+
+        self.stdout.write(f"Reading country data from {COUNTRY_CSV}...")
+        self.countries = build_country_map()
+
+        self.load_certs(filepath)
+
+    def get_country(self, value):
+        """Get Country object for incoming country value"""
+        value = ignore_null_str(value)
+        if value:
+            value = self.countries[value]
+            value = Country(code=value)
+        return value
+
+    def load_certs(self, cert_file):
+        self.counter = 0
+        self.test_excluded = 0
+        cert_list = []
+
+        reader = csv.DictReader(self.rectified_csv)
+        for row in reader:
+            self.counter += 1
+            if self.limit and self.counter > self.limit:
+                break
+
+            cert = self.make_certificate(row)
+            if cert:
+                cert_list.append(cert)
+        Certificate.objects.bulk_create(cert_list)
+
+        self.stdout.write(self.style.SUCCESS(
+            f'Processed {self.counter} certificate records.'))
+        self.stdout.write(self.style.SUCCESS(
+            f'Excluded {self.test_excluded} certificates with Licensee: TEST'))
+        self.stdout.write(self.style.SUCCESS(
+            f'Imported {self.counter-self.test_excluded} certificates!'))
+
+    def make_certificate(self, row):
+        """Parse CSV row into Certificate object"""
+        cert = Certificate()
+        cert.number = prepare_number(row['CertNumber'])
+
+        licensee_id = int(row['LicenseeID'])
+        # Ignore TEST(17) licensee records
```

Comment:
Found that this is likely not sufficient of a criteria.

Outside of ignoring certificates associated to a test licensee, can we also ignore certificates with numbers less than 5 digits?